packages/grafbase-sdk/src/config.ts

```diff
@@ -1,32 +1,54 @@
 import { AuthParams, Authentication } from './auth'
 import { CacheParams, GlobalCache } from './cache'
-import { GrafbaseSchema } from './grafbase-schema'
+import { GrafbaseFederationSchema, GrafbaseSchema } from './grafbase-schema'
 import { Experimental, ExperimentalParams } from './experimental'
 import { Federation, FederationParams } from './federation'
 
 /**
  * An interface to create the complete config definition.
  */
 export interface ConfigInput {
+  graph: GrafbaseSchema
```

Comment:
Why `GrafbaseSchema` instead of `Graph`?

Reply:
I didn't change the type names, just the user facing interface. We could also do that, but it's a noisier change which we can probably do separately

Reply:
The user can see the type name but otherwise it's internal mostly

Reply:
But let me take a quick look

Reply:
I would change it internally to avoid confusion. We are deprecating the schema property and adding a new graph property.

Reply:
Ok

Reply:
`Graph` is better, agreed.

Reply:
Done